content/blog/2020-MM-DD-treedata.table/index.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: "treedata.table"
+title: The treedata.table package
+package_version: 0.1.1
+author:
+  - April Wright
```

Comment:
We need [author template files](https://blogguide.ropensci.org/authortemplate.html) for Christian and Josef, and April should double check hers to make sure it's current (and perhaps add a photo?)